build.gradle

```diff
@@ -278,31 +279,8 @@ task ktlint(type: JavaExec, group: "verification") {
     outputs.dir("${project.buildDir}/reports/ktlint/")
 }
 
-task validateLocalizedMessages(group: "verification") {
-    doLast {
-        BufferedReader files = Files.newBufferedReader(Paths.get("${project.rootDir}/resources/resources/software/aws/toolkits/resources/localized_messages.properties"))
-        files
-            .lines()
-            .map({ item ->
-                if (item == null || item.isEmpty()) {
-                    return ""
-                }
-                String[] chunks = item.split("=")
-                if (chunks.length <= 1) {
-                    return ""
-                } else {
-                    return chunks[0]
-                }
-            })
-            .filter({ item -> !item.isEmpty() })
-            .reduce({ item1, item2 ->
-                if (item1 > item2) {
-                    throw new GradleException("localization file is not sorted:" + item1 + " > " + item2)
-                }
-
-                return item2
-            })
-    }
+tasks.register('validateLocalizedMessages', ValidateMessages) {
```

Comment:
do we want to keep the "verification" group identifier?

Reply:
Good catch I thought I readded that, fixed